mods/ra/rules/campaign-rules.yaml

```diff
@@ -35,3 +35,8 @@ E7.noautotarget:
 
 ^Vehicle:
 	Demolishable:
+
+^CivBuilding:
+	-SpawnActorOnDeath@1:
+	-SpawnActorOnDeath@2:
+	-SpawnActorOnDeath@3:
```

Comment:
I'm not sure if this is necessary. However, if we want to keep it that way, you should also remove `Explodes`.